src/main/java/com/danielagapov/spawn/Services/Event/EventService.java

```diff
@@ -155,4 +156,25 @@ public boolean deleteEventById(UUID id) {
     public List<UserDTO> getParticipatingUsersByEventId(UUID id) {
         return List.of();
     }
+
+    public UserParticipationStatus getUserParticipationStatus(UUID eventId, UUID userId) {
+
+        return UserParticipationStatus.invited;
+
+        // get list of all users at an event -> if no event, throw exception
+
+        // search EventUser for the participation status of given user -> no user, throw notInvited enum
+    }
+
+    public EventDTO inviteUser(UUID eventId, UUID userId) {
```

Comment:
```suggestion
    public void inviteUser(UUID eventId, UUID userId) {
```
This doesn't need to return anything, just throw a possible exception. Please implement this method as part of your PR.

Reply:
(Same with the `participateUser()` method, and their interface function signatures)

Reply:
i removed the participateUser method entirely as it has no real functionality. toggleParticipation, once implemented should do that job anyway. I think this allows us to use inviteUser more flexibly